airflow/models/dagbag.py

```diff
@@ -456,4 +456,5 @@ def sync_to_db(self):
         from airflow.models.dag import DAG
         from airflow.models.serialized_dag import SerializedDagModel
         DAG.bulk_sync_to_db(self.dags.values())
-        SerializedDagModel.bulk_sync_to_db(self.dags.values())
+        if self.store_serialized_dags:
+            SerializedDagModel.bulk_sync_to_db(self.dags.values())
```

Comment:
@kaxil FYI, plus the next hunk to models/s10n_dag.

Reply:
Means I can do

```
        non_serialized_dagbag = DagBag(store_serialized_dags=False, include_examples=False)
        non_serialized_dagbag.store_serialized_dags = True
        non_serialized_dagbag.sync_to_db()
```

Reply:
Nice